exec/java-exec/src/main/resources/drill-module.conf

```diff
@@ -112,7 +112,8 @@ drill.exec: {
     retry: {
       count: 7200,
       delay: 500
-    }
+    },
+    use.secure_acl: false
```

Comment:
This needs to be set to true in `/opt/mapr/drill/drill-1.11.0/conf/drill-distrib.conf` by `configure.sh` when security is enabled on a MapR cluster. Do we have a tracking work item for this?